pyccel/codegen/printing/cwrappercode.py

```diff
@@ -280,7 +285,7 @@ def _get_wrapper_name(self, used_names, func):
     # Functions managing  the creation of wrapper body
     # -------------------------------------------------------------------
 
-    def _valued_variable_management(self, variable, collect_var, tmp_variable):
+    def _valued_variable_management(self, variable, collect_var, tmp_variable, default_value):
```

Comment:
Can you please update the docstring?

Please note that the parameters are given in the wrong order there...

Reply:
Fixed 7364ac9f